x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -171,7 +251,18 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
           <EuiSpacer size="xs" />
         </>
       )}
-      {showSpikeAnalysisTable && (
+      {showSpikeAnalysisTable && !showUngrouped ? (
+        <SpikeAnalysisGroupsTable
```

Comment:
Would a link to Discover for the group level add any value - like `Show all docs with this set of common attributes`? Hard to tell with just the mock data.

<img width="715" alt="image" src="https://user-images.githubusercontent.com/7405507/189925905-2e09c03e-6191-444a-a3a1-5c61222cb75d.png">

Reply:
Sounds good for a follow up 👍